roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -21,7 +21,7 @@
         "/var/log/containers/nova:/var/log/containers/nova",
         "/var/lib/libvirt:/var/lib/libvirt",
         "/run/libvirt:/run/libvirt:shared",
-        "/var/lib/nova:/var/lib/nova:shared,z",
+        "/var/lib/nova:/var/lib/nova:shared",
```

Comment:
for context we were chattign about this a little on slack.

i think we may be able to proceed with this change
as it reverts to the tripleo bevhiaor

https://github.com/openstack-archive/tripleo-heat-templates/blob/stable/wallaby/deployment/nova/nova-compute-container-puppet.yaml#L1431

but its also only required for running nova on nfs which is a post ga feature.

now I'm not sure we want to have to deal with relableing the directory after ga or any issues that might introduce so I'm slightly inclined to merge this before ga to minimise risk.

however we need to ensure there are no other fallout form this first

Reply:
we need to retain this tripleo feature without bringing in a regression after GA.
So please let's merge, then deal to fallouts as it goes

Reply:
We can land this patch as edpm-ansible is also branched for GA already. So this commit will go to main and will be release post GA.
However if we want this in GA then we need:
* a Jira ticket that is an approved blocker of GA
* we need to cherry-pick this fix to the 18.0.0-proposed branch
* after the fix landed in 18.0.0-proposed make sure that the new edpm-ansible dependency is picked up by the upstream / downstream build.

Reply:
i agree with gibi.
we can proceed with merging this but if we want it in GA we need to follow the blockers process

this will not change the support status of /var/lib/nova on nfs

that will still be unsupported in GA and will be part fo a follow up release effectively as a tech preview.

but i also agree with bogdan that we don't want to do this after ga if we can avoid it

ill approve this now and file an issue tracker for ga and we can see if we get agreement to treat it as a blocker seperatly.